homework/index.js

```diff
@@ -30,15 +32,126 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
+  function renderContributors(contributors, contributorsContainer) {
+    createAndAppend('p', contributorsContainer, {
+      text: 'Contributors',
+      class: 'contributor-header',
+    });
+    contributors.forEach(contributor => {
+      const contributorList = createAndAppend('ul', contributorsContainer, {
+        class: 'contributor-list',
+      });
+
+      const contributorListItem = createAndAppend('li', contributorList);
+
+      const anchorLink = createAndAppend('a', contributorListItem, {
+        href: contributor.html_url,
+        target: '_blank',
+        class: 'contributor-item',
+      });
+
+      createAndAppend('img', anchorLink, {
+        src: contributor.avatar_url,
+        alt: contributor.login,
+        class: 'image',
+      });
+
+      const contributorDetails = createAndAppend('div', anchorLink, {
+        class: 'contributor-data',
+      });
+      createAndAppend('div', contributorDetails, { text: contributor.login });
+      createAndAppend('div', contributorDetails, {
+        text: contributor.contributions,
+        class: 'contributor-badge',
+      });
+    });
+  }
+
+  function fetchAndRenderData(selectedRepo, repoContainer, contributorsContainer, root) {
+    repoContainer.innerHTML = '';
+    contributorsContainer.innerHTML = '';
+
+    const updatedAt = new Date(selectedRepo.updated_at);
+
+    const repoTitles = { description: 'Description: ', forks: 'Forks: ', updated_at: 'Updated: ' };
+
+    const table = createAndAppend('table', repoContainer, { class: 'table' });
+    const tbody = createAndAppend('tBody', table);
+
+    const firstRow = createAndAppend('tr', tbody);
+    const leftCell = createAndAppend('td', firstRow);
+    const rightCell = createAndAppend('td', firstRow);
+
+    createAndAppend('span', leftCell, { text: 'Repository: ', class: 'repo-child left-cell' });
+    createAndAppend('a', rightCell, {
+      text: `${selectedRepo.name}`,
+      href: selectedRepo.html_url,
+      target: '_blank',
+      class: 'repo-child right-cell',
+    });
+
+    Object.keys(repoTitles).forEach(key => {
+      const tr = createAndAppend('tr', tbody);
+      const leftCell = createAndAppend('td', tr);
+      const rightCell = createAndAppend('td', tr);
```

Comment:
Line 95 and 96 produce ESLint errors:

```
95:13  error  'leftCell' is already declared in the upper scope   no-shadow
96:13  error  'rightCell' is already declared in the upper scope  no-shadow
```

This is because you have already used these names in an outer scope (lines 82-83) and using these same names makes the variable from the outer scope unreachable in the inner scope.  At this particular point, you have no use for the variables in the outer scope, but it is considered a bad practice anyway. 

A simple way to solve this is to declare the variables in lines 82 with `let` instead of `const` and to reuse those same variables here without `const` or `let`.

```js
let leftCell = createAndAppend('td', firstRow);  // line 82
... 
leftCell = createAndAppend('td', tr); // line 95
```